swig/python2/tests/NotificationTest.py

```diff
@@ -417,7 +417,7 @@ def test_notify_same_path(self):
          ["DELETED", "/ietf-interfaces:interfaces/interface[name='eth0']/ietf-ip:ipv4/address[ip='192.168.2.100']/ip"],
          ["DELETED", "/ietf-interfaces:interfaces/interface[name='eth0']/ietf-ip:ipv4/address[ip='192.168.2.100']/prefix-length"],
          ["DELETED", "/ietf-interfaces:interfaces/interface[name='eth0']/ietf-ip:ipv4/enabled"],
-         # ["DELETED", "/ietf-interfaces:interfaces/interface[name='eth0']/ietf-ip:ipv4/mtu"],
```

Comment:
Hi Lukas, 

Regarding the line commented out, tests fail if I change that.
When assertions are being checked, this one is missing from notifications but is expected.
Unfortunately, I haven't been able to figure out why is this specific notification missing. 
Test itself relies on https://github.com/sysrepo/sysrepo/blob/devel/tests/notifications_test_app.c but 
Maybe data for module changed since tests were first written.
All hints would welcome.
Thanks,
Antonio